scripts/sfpshow

```diff
@@ -280,8 +280,26 @@ class SFPShow(object):
                                                 units)
         return output
 
-    # Convert sfp info in DB to cli output string
-    def convert_sfp_info_to_output_string(self, sfp_info_dict):
+    # Convert cmis sfp info in DB to cli output string
+    def convert_cmis_sfp_info_to_output_string(self, sfp_info_dict):
```

Comment:
Looks like, convert_cmis_sfp_info_to_output_string and convert_non_cmis_sfp_info_to_output_string have quite some code in common.
would it better to combine them into one function to reuse the code more?
something like:
```
    def convert_sfp_info_to_output_string(self, sfp_info_dict):
        indent = ' ' * 8
        output = ''
        
        data_map = CMIS_DATA_MAP if is_cmis else QSFP_DATA_MAP
        sorted_data_map_keys = sorted(data_map, key= data_map.get)
        for key in sorted_data_map_keys:
            if key == 'cable_type':
                output += '{}{}: {}\n'.format(indent, sfp_info_dict['cable_type'], sfp_info_dict['cable_length'])
            elif key == 'cable_length':
                pass
            elif key == 'specification_compliance':
                if not is_cmis:
                     ....
                else:
                     ...
            elif key == 'application_advertisement':
                output += covert_application_advertisement_to_output_string(indent, sfp_info_dict)
            else:
                output += '{}{}: {}\n'.format(indent, data_map[key], sfp_info_dict[key])

        return output
```

Reply:
Yes, addressed this now